src/lib/cpl.js

```diff
@@ -0,0 +1,97 @@
+/* @flow */
```

Comment:
general question, do these events need to be logged to Cal (`.info`, `.warning`, `.error`) or could they be sent via `.track` only?

I think right now `.track` will also log to Cal